core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -136,14 +145,49 @@ var ExplorationEditorTranslationTab = function() {
 
   var solutionTabText = element(by.css('.protractor-test-solution-text'));
 
-  var numericalStatus = element(
+  this.numericalStatus = element(
     by.css('.protractor-test-translation-numerical-status'));
 
   var _selectLanguage = function(language) {
     element(by.css('.protractor-test-translation-language-selector')).
       element(by.cssContainingText('option', language)).click();
   };
 
+  this.expectSaveUploadedAudioButtonToBeDisabled = function() {
+    expect(saveUploadedAudioButton.getAttribute('disabled')).toBe('true');
+  };
+
+  var audioUploadInputElem = element(by.className(
```

Comment:
Move all variable declaration to the top of the file in sorted order. (Ditto else.)

Reply:
done